### PR TITLE
Feature/63 optimize category mapping for children

### PR DESCRIPTION
**Pull request by jprotin**

### CODE REVIEW

**Comment by kylecrum** [comment]
@jprotin why are there 160 files changed and 700,000 new lines of code?

**Comment by jprotin** [comment]
Hi @kylecrum, normally no ... I have not checked this, we have 10 files max. Ah! I know why, in my local I changed the file access rights and git takes it as an update ...

File: src/reverb/reverb.php [hunk 1]
```diff
@@ -789,6 +783,81 @@ protected function postProcess()
...
+                'reverb_slug' => array(
+                    'title' => '',
+                    'search' => false,
+                    'orderby' => false,
+                )
+            );
+            //=========================================
+            //         GET DATAS FOR LIST
+            //=========================================
+            $datas = $this->reverbSync->getListProductsWithStatus($fields_list, true, false);
```
Comment (kylecrum) [inline src/reverb/reverb.php]: what does this do? Does this really try to get all of these listings from reverb via the API?

Comment (jprotin) [reply src/reverb/reverb.php]: Hi @kylecrum 
no it does not do that. This function gets in the PrestaShop database all products eligible to the listing Sync. No call Reverb api

Comment (kylecrum) [reply src/reverb/reverb.php]: 🥇

**Pull request merged**